crates/fluvio-controlplane-metadata/src/smartmodule/spec.rs

```diff
@@ -104,6 +106,40 @@ pub struct SmartModuleWasmSummary {
     pub wasm_length: u32,
 }
 
+pub struct WasmBytes(Vec<u8>);
```

Comment:
Provide a new type to differ from the implementation of `Encoder` for `Vec<T>`.

Reply:
Since this is so useful let's move to into `fluvio-protocol-core`.